coderd/database/dbmem/dbmem.go

```diff
@@ -748,6 +748,22 @@ var deletedUserLinkError = &pq.Error{
 	Routine: "exec_stmt_raise",
 }
 
+// m1 and m2 are equal iff |m1| = |m2| ^ m1 ⊆ m2
+func tagsEqual(m1, m2 map[string]string) bool {
+	return len(m1) == len(m2) && tagsSubset(m1, m2)
+}
+
+// m2 is a subset of m1 if each key in m1 exists in m2
+// with the same value
```

Comment:
Isn't this the other way around, m1 is a subset of m2. At least, that's what I think the code says, as does the notation for `tagsEqual`.